### DISCUSSION TITLE
Tiers and pricing of plan "Organization" after end of beta phase

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#3832] [Q&A]
I'm a bit confused about how the tiers and their pricing will work. 

The description says an organization is at least 2 members and it's 16€ per member. Also it says "pay only for additional active users". So does the base price include the first two members? 
Or is it at least 32€ with every member costing 16€?

And am I understanding it correctly, that a collaborator on an organizations private project needs to be a member of the organization?

So in my current case I have a project hosted owned by my organization and I need 4 mobile devices to be able to simultaneously work on it.  How many instances of which tier would I need for that?

**Comment by suricactus** [comment]
Hi, @chrysenteron . Sorry for the confusion. It's 16 per user and you have to pay for at least 2 users, which means at least 32. This is the amount that you can never get lower.

As soon as there are more than 2 users active during this month (downloading or uploading data), you pay extra 16 for each of them. Let me try to explain with 5 examples :

| number of organization members | number of active users |  amount to pay |
|---------------------------------------------|---------------------------------|---------------------|
| 4 | 0 | 32 |
| 4 | 1 | 32 |
| 4 | 2 | 32 |
| 4 | 3 | 48 |
| 4 | 4 | 64 |

Hope it is clear, let us know if we need to clarify something else.

**Reply by chrysenteron** [reply]
Thanks for your response. I have one more question though: Is there some amount of data hosting included? Or is it also at least an additional 5€ per month?

**Reply by chrysenteron** [reply]
@suricactus Also, how exactly does yearly billing differ from monthly billing? Is the total price still calculated based on active users each month?

**Reply by chrysenteron** [reply]
@suricactus  Would be great if I could get an Answer to this soon, since we'll need some time to discuss and coordinate the switch to a paid subscription in our team. Thanks in advance :)

**Reply by suricactus** [reply]
Hey, @chrysenteron . Sorry for the delayed answer, you might have seen we had our priorities rapidly shifted in the second half the week. 

Let me try to answer your questions:

> Is there some amount of data hosting included? Or is it also at least an additional 5€ per month?

Organizations have no data hosting included for now, they need to pay at least 5€ per month. Any additional GB is fixed 5€ each. If you use 10GB, then it's 50€ per month.

> Also, how exactly does yearly billing differ from monthly billing? 

Being an early bird (or :bee: ) will automatically apply a discount for the next 12 months for you.

> Is the total price still calculated based on active users each month?

Yes, you get billed every month.
